exporters/prometheus/config.go

```diff
@@ -57,3 +58,18 @@ func WithRegisterer(reg prometheus.Registerer) Option {
 		return cfg
 	})
 }
+
+// WithoutUnits disables exporter's addition of unit suffixes to metric names,
+// and will also prevent unit comments from being added in OpenMetrics once
+// unit comments are supported.
+//
+// By default, metric names include a unit suffix to follow Prometheus naming
+// conventions. For example, the counter metric request.duration, with unit
+// milliseconds would become request_duration_milliseconds_total.
```

Comment:
```suggestion
// milliseconds would become request_duration_milliseconds.
```

Reply:
oh, this is a bug.  We need to add `_total` to counters.  Thanks for pointing that out.

Reply:
https://github.com/open-telemetry/opentelemetry-go/issues/3359